schema/network/network.go

```diff
@@ -60,6 +62,8 @@ func (n Network) Source() Source {
 		return EthereumSource
 	case Farcaster:
 		return FarcasterSource
+	case Mastodon:
```

Comment:
Maybe it could be `ActivityPubSource` because all these platforms follow the same AP protocol?